spec/support/pages/holidays.rb

```diff
@@ -12,6 +12,8 @@ class Holidays < Base
       element :content, '.fc-content'
     end
 
+    section :calendar, Pages::CalendarSection, '.t-calendar'
```

Comment:
Makes no sense for a section to be namespaced within pages. Also suffixing the section with `Section` would not be necessary if it was namespaced within `Sections`.